### ISSUE TITLE
Please remove flash elements

### ISSUE THREAD

**Issue by trentonknew** [sivann/itdb#70]
If at all possible, would it be hard to remove the flash elements from the menus?  Flash support is dying a low and painful death.

**Comment by nikband** [comment]
Yes I hope this is for new ITDB there is someone can convert Flash element into html5 or similar?

**Comment by PoerOne** [comment]
May you can chekout [https://github.com/PoerOne/itdb](url)

**Comment by PoerOne** [comment]
May you can check out on my repo [https://github.com/PoerOne/itdb/tree/patch-1](url)

Hope help.
